x/fees/client/cli/query.go

```diff
@@ -132,15 +133,15 @@ func GetCmdQueryParams() *cobra.Command {
 	return cmd
 }
 
-// GetCmdQueryDeployerFees implements a command that returns all
-// contracts that a deployer has registered for fee distribution
+// GetCmdQueryDeployerFees implements a command that returns all contracts that
+// a deployer has registered for fee distribution
 func GetCmdQueryDeployerFees() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "fee-infos-deployer [address]",
+		Use:     "fees-deployer [address]",
 		Args:    cobra.ExactArgs(1),
-		Short:   "Query all contracts that a deployer has registered.",
-		Long:    "Query all contracts that a deployer has registered for fee distribution.",
-		Example: fmt.Sprintf("%s query fees fee-infos-deployer <address>", version.AppName),
+		Short:   "Query all fees that a deployer has registered",
```

Comment:
Not sure, if `fees` is really a better replacement for `contracts` - they are contracts, that are registered after all? 

I guess it refers to the fact, that the actual `Fee` types are queried, but this might not be relevant to the CLI user and might be less clear what the command does?

Reply:
@MalteHerrmann I think it's unclear because the term `Fee` is too broad (as you mentioned above). We should improve this term and then the description will make more sense, i.e. 		`Short:   "Query all feesplits that a deployer has registered",`

Reply:
Sounds good!